app/javascript/shared/components/useKeyboardShortcuts.js

```diff
@@ -126,10 +126,13 @@ export function useKeyboardShortcuts(
     const keyEvent = (e) => {
       if (e.defaultPrevented) return;
 
-      // Get special keys
-      const keys = `${e.ctrlKey || e.metaKey ? 'ctrl+' : ''}${
-        e.altKey ? 'alt+' : ''
-      }${(e.ctrlKey || e.metaKey || e.altKey) && e.shiftKey ? 'shift+' : ''}`;
+      const ctrlKeyEntry = e.ctrlKey ? 'ctrl+' : '';
```

Comment:
We don't technically need these as variables, but I think the way we handle the chained keys makes it important for DX that we can easily see what order those keycodes are expected in, and this - to me - made it more readable